applications/nfc/nfc_worker.c

```diff
@@ -636,7 +636,7 @@ void nfc_worker_emulate_mifare_ul(NfcWorker* nfc_worker) {
                             break;
                         }
                     } else {
-                        FURI_LOG_W(TAG, "Not valid command: %02X", rx_buff[0]);
+                        FURI_LOG_D(TAG, "Not valid command: %02X", rx_buff[0]);
```

Comment:
I propose to completely remove logging from this function and leave a comment about the side effects of calling a blocking function.

Reply:
Done.